src/components/bcourses/roster/RosterPhotos.vue

```diff
@@ -22,6 +24,7 @@
           <RosterPhoto
             :on-load="() => student.hasRosterPhotoLoaded = true"
             :student="student"
+            :photos-per-page="photosPerPage ? 1 : all"
```

Comment:
@mlincoln - Minor point: we try to keep component attributes (ie, props) in alpha order.